src/badges/templates/badges/badge_confirm_delete.html

```diff
@@ -5,7 +5,7 @@
 
 {% block content %}
 <form action="" method="post">{% csrf_token %}
-  <p>Are you sure you want to Delete this Achievement?  All rewardings will be deleted.<p>
+  <p>Are you sure you want to Delete this Achievement? All instances of "{{object.name}}" badge will be deleted. Therefore, this badge will be removed from all users.<p>
```

Comment:
Change to "Are you sure you want to delete this badge? This badge will be removed from all users that have earned it.  This action is irreversible."